### DISCUSSION TITLE
The original LoveIt theme is back!

### DISCUSSION THREAD

**Discussion by HEIGE-PCloud** [HEIGE-PCloud/DoIt#609] [Polls]
After being inactive for a long time, the author of the original LoveIt theme has come back! The LoveIt theme is being actively maintained again. 

https://github.com/dillonzq/LoveIt/issues/672

The DoIt theme is a fork of the LoveIt theme and is maintained by me and all community members for around a year. A lot of new features are added but the overall design is kept the same. There is also a huge [1.0 release](https://github.com/HEIGE-PCloud/DoIt/pull/571) under active development with tons of improvements, features and optimization.

However, I am a bit unsure about what to do with this project next. There are a few options and I am willing to hear your ideas.

1. **Keep developing as usual.**
This is the easiest way to go. DoIt will "compete" with the LoveIt theme. The LoveIt theme has much greater influence, but the DoIt theme has more features and improvements. This competitive relationship can potentially help both themes to grow better. However, both themes have a very similar looking and it is probably confusing for new users to understand the complex history and relationship between these two very "similar-looking" themes. This approach also wastes quite a lot of development time as both themes are building their own wheels separately.

2. **Merge DoIt to LoveIt (somehow)?**
Potentially, two themes can be merged together. A new organization can be created and all users can be directed to the "merged" theme. Therefore all contributors can work on the same project, and all users can have all features and improvements, and the probability of the theme being abandoned again is much lower.
However, this depends on whether @dillonzq  is willing to do so, and the technical challenges behind the merging process cannot be overlooked. I personally will have less "control" on the design and implementation of the theme as well.

What do you think? 🤔

**Comment by dillonzq** [comment]
Thanks for your excellent work on DoIt theme. I support that this theme continues to be developed according to your plan. Since the original LoveIt theme had already generated a few commits, there are some differences between them that it's difficult to merge them together. At the same time I already have some ideas for refactoring the LoveIt theme, which will have some impact on merging the two themes.
I found that this theme supports a lot of exciting features under your maintenance, such as more search options and more comment systems. I am happy to gradually migrate to the original LoveIt theme. Of course, if you are willing, any pull requests are welcome.

**Comment by wey-gu** [comment]
> After being inactive for a long time, the author of the original LoveIt theme has come back! The LoveIt theme is being actively maintained again. 
> 
> 
> 
> https://github.com/dillonzq/LoveIt/issues/672
> 
> 
> 
> The DoIt theme is a fork of the LoveIt theme and is maintained by me and all community members for around a year. A lot of new features are added but the overall design is kept the same. There is also a huge [1.0 release](https://github.com/HEIGE-PCloud/DoIt/pull/571) under active development with tons of improvements, features and optimization.
> 
> 
> 
> However, I am a bit unsure about what to do with this project next. There are a few options and I am willing to hear your ideas.
> 
> 
> 
> 1. **Keep developing as usual.**
> 
> This is the easiest way to go. DoIt will "compete" with the LoveIt theme. The LoveIt theme has much greater influence, but the DoIt theme has more features and improvements. This competitive relationship can potentially help both themes to grow better. However, both themes have a very similar looking and it is probably confusing for new users to understand the complex history and relationship between these two very "similar-looking" themes. This approach also wastes quite a lot of development time as both themes are building their own wheels separately.
> 
> 
> 
> 2. **Merge DoIt to LoveIt (somehow)?**
> 
> Potentially, two themes can be merged together. A new organization can be created and all users can be directed to the "merged" theme. Therefore all contributors can work on the same project, and all users can have all features and improvements, and the probability of the theme being abandoned again is much lower.
> 
> However, this depends on whether @dillonzq  is willing to do so, and the technical challenges behind the merging process cannot be overlooked. I personally will have less "control" on the design and implementation of the theme as well.
> 
> 
> 
> What do you think? 🤔

Sorry just saw this now, I will support any of your choices.

I like doit to be as-is but if you decide to push it to other directions, I will support, too.❤️

**Comment by CXwudi** [comment]
And one reason I continue to use DoIt is because of the Waline support 😂. LoveIt is still using Valine

**Comment by ghost** [comment]
Thanks for your willing work. I chose DoIt because of pjax support and hope the project will continue.

**Comment by CXwudi** [comment]
Not to mentioned that by today, the LoveIt theme stopped again in July
